resources/builder.inc.sh

```diff
@@ -1346,12 +1346,11 @@ _builder_parse_expanded_parameters() {
   fi
 
   if builder_is_debug_build; then
-    readonly BUILDER_CONFIGURATION=debug
+    BUILDER_CONFIGURATION=debug
   else
-    readonly BUILDER_CONFIGURATION=release
+    BUILDER_CONFIGURATION=release
```

Comment:
Sadly these cannot be `readonly`, but only because of to the test script which calls `builder_parse` multiple times.